apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -334,6 +345,11 @@ else
     || test_exit_code=$?
 fi
 
+if [[ "$reuse_simulator" == false ]]; then
+  xcrun simctl shutdown "$simulator_id"
```

Comment:
this might not hit if the test times out, maybe nothing we can do safely in the creator script

Reply:
And AFAIK that's generally how xctesrunner worked before anyway, right?